NARFO_BE/NARFO_BE/Models/DedicatedStatus.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace NARFO_BE.Models
+{
+    public partial class DedicatedStatus
+    {
+        public int DedicatedId { get; set; }
+        public string MemNo { get; set; }
+        public string Dstype { get; set; }
+        public string Dsno { get; set; }
+        public DateTime? ApplicationDate { get; set; }
+        public string Approved { get; set; }
+        public DateTime? DateApproved { get; set; }
+        public string Motivation { get; set; }
+        public string RefusalReason { get; set; }
+        public string Termination { get; set; }
+        public string TerminationReason { get; set; }
+        public string TrsfAssociation { get; set; }
+        public decimal? TrsfAssociationFarn { get; set; }
+        public string TrasfDsno { get; set; }
```

Comment:
this is a bunch of giberish. create a description describing what this actually means